litani

```diff
@@ -642,7 +563,20 @@ async def init(args):
 
 
 async def add_job(args):
-    await lib.add_job.add_job(vars(args))
+    await lib.jobs.add_job(vars(args))
+
+
+async def set_jobs(args):
```

Comment:
Oh wait, it looks like you have an identical implementation of this in `lib.jobs`?